lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -272,10 +274,10 @@ class QcFlagService {
      * Delete single instance of QC flag
      * @param {number} id QC flag id
      * @param {object} [relations] QC Flag entity relations
-     * @param {UserIdentifier+IsAdminOption} [relations.user] user
+     * @param {UserWithRoles} [relations.user] user
```

Comment:
I think those should be mandatory

Reply:
https://github.com/AliceO2Group/Bookkeeping/pull/1501#discussion_r1562298591

Reply:
If I call a function that, in its signature, allow me to omit a parameter, I do not expect it to throw because I omited the parameter, it makes no sense.
If you expect the user to be specified, it should be explicit in the documentation.

Reply:
made mandatory